frontend/src/app/patients/__snapshots__/EditPatient.test.tsx.snap

```diff
@@ -1047,7 +1019,25 @@ Object {
                           class="usa-radio__label"
                           for="35-val-not_hispanic"
                         >
-                          Not Hispanic
+                          No
+                        </label>
+                      </div>
+                      <div
+                        class="usa-radio"
+                      >
+                        <input
+                          class="usa-radio__input"
+                          data-required="false"
+                          id="35-val-refused"
+                          name="ethnicity"
+                          type="radio"
+                          value="refused"
+                        />
+                        <label
+                          class="usa-radio__label"
+                          for="35-val-refused"
+                        >
+                          Prefer not to answer
```

Comment:
I think this snapshot needs to be updated since you removed "Prefer not to answer" for now

Reply:
Done in 7e4b0f6.